grunttasks/htmllint.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('htmllint', {
+    options: {
+      ignore: [
+        'Bad value “X-UA-Compatible” for attribute “http-equiv” on XHTML element “meta”.',
+        'Consecutive hyphens did not terminate a comment. “--” is not permitted inside a comment, but e.g. “- -” is.',
+        'Duplicate ID “-”.',
+        'Text run is not in Unicode Normalization Form C.',
+        'The first occurrence of ID “-” was here.',
+        'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+        'The document is not mappable to XML 1.0 due to a trailing hyphen in a comment.',
+        'Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.',
+        'Bogus comment.'
+      ]
+    },
+
+    dist: [
+      '<%= yeoman.page_template_dist %>/*/*.html',
+      '!<%= yeoman.page_template_dist %>/it_CH/*.html',
+      '!<%= yeoman.page_template_dist %>/{privacy,terms}/*.html'
+    ],
+
+    all: [
+      '<%= yeoman.app %>/**/*.{html,mustache}',
```

Comment:
I tried running `grunt htmllint:all` and it seemed to have a bit of sadness:

```
...
app/scripts/templates/pp.mustache [L1:C37] Start tag seen without seeing a doctype first. Expected “<!DOCTYPE html>”.
app/scripts/templates/pp.mustache [L1:C37] Element “head” is missing a required instance of child element “title”.
app/scripts/templates/pp.mustache [L10:C29] Article lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all articles.
app/scripts/templates/ready.mustache [L1:C36] Start tag seen without seeing a doctype first. Expected “<!DOCTYPE html>”.
app/scripts/templates/ready.mustache [L1:C36] Element “head” is missing a required instance of child element “title”.
app/scripts/templates/reset_password.mustache [L1:C36] Start tag seen without seeing a doctype first. Expected “<!DOCTYPE html>”.
app/scripts/templates/reset_password.mustache [L1:C36] Element “head” is missing a required instance of child element “title”.
app/scripts/templates/settings.mustache [L1:C36] Start tag seen without seeing a doctype first. Expected “<!DOCTYPE html>”.
app/scripts/templates/settings.mustache [L1:C36] Element “head” is missing a required instance of child element “title”.
app/scripts/templates/settings.mustache [L20:C6] Text not allowed in element “ul” in this context.
app/scripts/templates/settings.mustache [L24:C6] Text not allowed in element “ul” in this context.
app/scripts/templates/settings.mustache [L31:C8] Text not allowed in element “ul” in this context.
app/scripts/templates/settings.mustache [L35:C2] Text not allowed in element “ul” in this context.
app/scripts/templates/settings.mustache [L35:C12] End tag “section” seen, but there were open elements.
app/scripts/templates/settings.mustache [L18:C22] Unclosed element “ul”.
app/scripts/templates/settings/avatar.mustache [L1:C36] Start tag seen without seeing a doctype first. Expected “<!DOCTYPE html>”.
...
>> 928 files checked, 219 errors in 34 files
Warning: Task "htmllint:all" failed. Use --force to continue.

Aborted due to warnings.


Execution Time (2015-07-07 19:54:22 UTC)
loading tasks   1.7s  ▇▇▇▇▇ 7%
htmllint:all   22.7s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 93%
Total 24.4s
```

Oddly, all 219 errors seemed to be in *.mustache files. (and some even seem to be actual HTML errors, like a missing `</ul>` in the /settings page.)